### ISSUE TITLE
Have option for change volume?

### ISSUE THREAD

**Issue by octaviz** [SeydX/homebridge-bravia-tvos#52]
Before installing it I wanted to know if you have options to change the volume of the TV

**Comment by pbendersky** [comment]
I haven't seen it documented on the README, but when you are within the Remote Control on your iOS device, the hardware volume keys will control the TV device.

**Comment by octaviz** [comment]
Hi pbendersky!, I have already tried that option to do it with the iOS Remote App, my Sony TV does not work for me, I do not know if it is because it is a little old (2015) with Android, it is thinking and there is no way. In the end I decided to use the 2.5.2 de SeydX version that works well to control the volume level directly with a button without using the Apple remote.

v2.5.2 https://github.com/SeydX/homebridge-sonybravia-platform

Thank you very much friend!
